### PR TITLE
Stats command

### PR DESCRIPTION
**Pull request by jeanleonov**
Repeats https://github.com/AppScale/appscale-tools/pull/632.
Add stats command to print node, process and proxy statistics.

### CODE REVIEW

**Comment by jeanleonov** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4721/

File: appscale/tools/utils.py [hunk 1]
```diff
@@ -177,3 +177,80 @@ def mkdir(dir_path):
...
+
+  # Prepare xterm prefix to prepend it to the text
+  try:
+    styles_prefix = u"".join((STYLES_MAP[style] for style in marks))
+  except KeyError as err:
+    raise UnknownStyle("Unknown style ({err}). Allowed are: {styles}"
+                       .format(err=err, styles=STYLES_MAP.keys().sort()))
+
+  # Resolve collisions with styles of wrapped text
+  text = text.replace(u"\x1b[0m", u"\x1b[0m{}".format(styles_prefix))
```
Comment (cdonati) [inline appscale/tools/utils.py]: Why not use the termcolor module?

Comment (jeanleonov) [reply appscale/tools/utils.py]: Termcolor doesn't handle wrapping of wrapped text. It puts `u"\x1b[0m"` in the end of styled string, and it resets all styles.. But I want to be able to make bold line which contains some red words.

**Comment by jeanleonov** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/5079/

File: appscale/tools/appscale_stats.py [hunk 2]
```diff
@@ -0,0 +1,655 @@
...
+      continue
+
+    if service_name_id not in unique_proxies:
+      summary_proxy = unique_proxies[service_name_id] = defaultdict(int)
+    else:
+      summary_proxy = unique_proxies[service_name_id]
+
+    summary_proxy["servers_count"] = node["servers_count"]
+    servers_down = sum(1 for s in node["servers"] if s["status"] != "UP")
+    summary_proxy["servers_down_count"] = servers_down
```
Comment (scragraham) [inline appscale/tools/appscale_stats.py]: Combine line 564 + 563. 
summary_proxy["servers_down_count"] = sum(1 for s in node["servers"] if s["status"] != "UP")

Comment (jeanleonov) [reply appscale/tools/appscale_stats.py]: @scragraham the combination of two lines has 96 signs which exceeds our agreement (80 signs).
I have doubts if splitting it as 
```
    summary_proxy["servers_down_count"] = sum(
      1 for s in node["servers"] if s["status"] != "UP")
```
or other ways without introducing intermediate variable would have better readability.

Comment (scragraham) [reply appscale/tools/appscale_stats.py]: you can wrap the if stmt of the comprehension:
```
    summary_proxy["servers_down_count"] = sum(1 for s in node["servers"]
                                              if s["status"] != "UP")

````
Also: the world won't come to an end if you exceed 80 characters (I think)

**Comment by jeanleonov** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/5106/

**Pull request merged**